modules/@apostrophecms/schema/ui/apos/components/AposInputRange.vue

```diff
@@ -0,0 +1,248 @@
+<template>
+  <AposInputWrapper
+    :modifiers="modifiers" :field="field"
+    :error="effectiveError" :uid="uid"
+  >
+    <template #body>
+      <div class="apos-input-wrapper">
+        <div class="apos-range-input">
+          <input
+            type="range"
+            :min="field.min"
+            :max="field.max"
+            :step="field.step"
+            class="apos-range"
+            v-model="next"
+            :id="uid"
+          >
+          <div class="apos-range-input__scale">
+            <span class="apos-range-input__scale-value apos-range-input__scale-value--min">
+              {{ minLabel }}
+            </span>
+            <span class="apos-range-input__scale-value apos-range-input__scale-value--max">
+              {{ maxLabel }}
+            </span>
+          </div>
+        </div>
+        <div class="apos-range-value" :class="{'is-unset': !isSet}">
+          {{ valueLabel }}
```

Comment:
I'm wondering if this should be hidden from AT. The input itself will announce the value and then you don't need more labeling to distinguish from the min/max values.